recipes/google-cloud-container-feedstock/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "google-cloud-container" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 566834ef43e79917b112e3bd2848e84cfb0f4d7b565581607e2548f5c8e5d87a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - google-api-core>=1.6.0,<2.0.0dev
```

There needs to be a space here between the package name and the version constraint:
```suggestion
    - google-api-core >=1.6.0,<2.0.0dev
```
(and the same for the others)

---

Space inserted between package and version in all requirement sections in all recipe meta.yaml's.